### DISCUSSION TITLE
Regular expression pattern remover - what is it

### DISCUSSION THREAD

**Discussion by N3UE** [winder/Universal-G-Code-Sender#2325] [Help]
the Regular expression pattern remover ...
So I figured how to add this... "[Mm]7" to remove the M7 that Easel puts in the gcode.
But can anyone explain to me just what the "Regular expressions" are and how to use them?
I searched but found nothing.

**Comment by bLouChip** [comment]
https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference
